docs/content/stable/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-switchover.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Planned Switchover with disaster recovery
+headerTitle: Planned switchover
+linkTitle: Switchover
+description: Switchover using disaster recovery
+headContent: Switch application traffic to the DR replica
+menu:
+  stable_yugabyte-platform:
+    parent: disaster-recovery
+    identifier: disaster-recovery-switchover
+    weight: 40
+type: docs
+---
+
+Planned switchover is the process of switching write access from the DR primary to the DR replica without losing any data (zero RPO). Planned switchover is typically performed during a maintenance window.
+
+Switchover can be used by enterprises when performing regular business continuity or disaster recovery testing. Switchover can also used for failback purposes. That is, after a [Failover followed by a Repair DR](../disaster-recovery-failover/), you can use switchover to switch traffic back to the original DR primary after it is brought back online.
+
+## Perform switchover
+
+First, confirm there is no excessive lag between the DR primary and replica. You can monitor lag on the **Disaster Recovery** tab.
```

Comment:
Add this link - https://deploy-preview-20625--infallible-bardeen-164bc9.netlify.app/stable/yugabyte-platform/back-up-restore-universes/disaster-recovery/disaster-recovery-setup/#monitor-replication to see the screenshot to "monitor lag"